src/components/common/settings/KeyWords.tsx

```diff
@@ -20,12 +21,17 @@ import SaveButton from './SaveButton';
 
 const ENTER_KEY = 'Enter';
 
+type keywordsResourceData = { keywords: string[] };
+
 const KeyWords: FC = () => {
   const [word, setWord] = useState('');
   const [keywordsList, setKeyWordsList] = useState<string[]>([]);
   const { patchAppSetting, postAppSetting, appSettingArray } =
     useAppSettingContext();
 
+  const areEqual = (arr1: string[], arr2: string[]): boolean =>
+    arr1.length === arr2.length && arr1.every((s, i) => s === arr2[i]);
```

Comment:
You could use `lodash.isEqual` function for this. It can be imported as a standalone package [see the docs](https://lodash.com/docs/4.17.15#isEqual) and [the package on npm](https://www.npmjs.com/package/lodash.isequal)

Reply:
Est-ce que je dois l'installer comme j'avais installé le random generator de couleur ?